.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules/
 dist/
-.env
\ No newline at end of file
+.env
+backend/prisma/migrations/
```

Comment:
we probably shouldn't be ignoring all migrations! these will need to be run by everyone in order to keep everyone's dbs up to date